packages/bitcore-node/src/modules/ethereum/p2p/parityRpc.ts

```diff
@@ -91,7 +91,7 @@ export class ParityRPC {
 
   public send<T>(data: JsonRPCRequest) {
     return new Promise<T>((resolve, reject) => {
-      this.web3.eth.currentProvider.send(data, function(err, data) {
+      this.web3.eth.currentProvider!['send'](data, function(err, data) {
```

Comment:
`this.web3.eth.currentProvider` type is `provider`

```
    | HttpProvider
    | IpcProvider
    | WebsocketProvider
    | string
    | null;
```

The `!` gets rid of `| null`

Each Provider has a `send()` property:
ex. `WebsocketProvider.send(), HttpProvider.send(), ...`

`string` does not have a send property so `['send']`

Reply:
https://github.com/ethereum/web3.js/blob/043cb81d40ede0f6ef1889222c5671f9678e409e/packages/web3-core/types/index.d.ts#L413

Reply:
lets do 
```
let provider = this.web3.eth.currentProvider as HttpProvider;
provider.send(data
```